README.md

```diff
@@ -1,2 +1,35 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Flowchart : https://i.gyazo.com/4f236f8ca944556c4a966e149faec5b7.png
```

Comment:
Flowchart:
- Always add arrows to the lines in a flowchart, to make the direction of flow clear.
- The mechanic whereby the game ends after the number of guesses exceeds a limit is not present in the flowchart.
- Otherwise, flowchart looks OK.